cylc/rose/__init__.py

```diff
@@ -113,12 +128,34 @@
    +       [[Hello_Mars]]
 
 
+.. _rose-stem:
+
 Rose Stem
 =========
 
+.. seealso::
+
+   `Rose Stem documentation
+   <https://metomi.github.io/rose/2019.01.2/html/tutorial/rose/furthertopics/
+   rose-stem.html>`_
```

Comment:
I didn't know we could break lines in links this this. #TIL

Reply:
I didn't either, 'till I tried it!